iothub/device/src/Common/Security/SharedAccessSignatureBuilder.cs

```diff
@@ -33,10 +45,20 @@ public string Key
             }
         }
 
+        /// <summary>
+        /// The resource being accessed
```

Comment:
Probably resource Id? An example might help here. Ex: iothubname/devices/deviceId

Reply:
Preferred notation for **example** would be E.G.